tooling/build/package.json

```diff
@@ -1,5 +1,15 @@
 {
   "name": "build",
   "version": "0.0.0",
-  "private": true
-}
\ No newline at end of file
+  "private": true,
+  "scripts": {
+    "format": "prettier --check . --ignore-path ../../.gitignore",
+    "format:fix": "prettier --write . --ignore-path ../../.gitignore",
+    "lint": "eslint",
+    "lint:fix": "eslint --fix"
+  },
+  "devDependencies": {
+    "@isomer/eslint-config": "*",
+    "@isomer/prettier-config": "*"
+  }
```

Comment:
nit: might need an extra 

```
  "prettier": "@isomer/prettier-config"
```

so formatting works properly

Reply:
Added in 2e49116ea563bf870418089a4162e584403c1e3c.